sktime/detection/skchange/moving_window.py

```diff
@@ -0,0 +1,21 @@
+"""Moving Window detector."""
+
+from sktime.detection.base import BaseDetector
+from sktime.utils.dependencies import _placeholder_record
+
+
+@_placeholder_record(
+    "skchange.change_detectors.moving_window", dependencies="skchange>=0.6.0"
```

Comment:
Is there any direction on whether explicitly specifying the `skchange` dependency is necessary? If so, what should be the version? How would we handle the case where user has a version mismatch the sktime placeholder is imported instead?